### ISSUE TITLE
How to check for the expiry of the RefreshToken?

### ISSUE THREAD

**Issue by Sunita76** [FortnoxAB/csharp-api-sdk#269]
For checking the expiry of the AccessToken I can use:
var auth = new StandardAuth(accessToken);
            var isExpired = auth.IsExpired();
Can I check the RefreshToken expiry also in the same way?

Thanks

**Comment by richardrandak** [comment]
No, the refresh token does not have the JWT format which could be decoded in order to retrieve its expiration time. Since the SDK does not know when the refresh token was obtained, there is no way to calculate the expiration time either.

So, you have implement your own way to manage the refresh token expiration time.

**Comment by Sunita76** [comment]
Thanks a lot for the explanation.
